clang/include/clang/CIR/Dialect/IR/CIRStdOps.td

```diff
@@ -61,5 +61,11 @@ def CIR_IterBeginOp: CIR_StdOp<"begin",
 def CIR_IterEndOp: CIR_StdOp<"end",
   (ins CIR_AnyType:$container),
   (outs CIR_AnyType:$result)>;
+def CIR_StdVectorCtorOp: CIR_StdOp<"vector_cxx_ctor",
```

Comment:
I'd like to see examples here in some form. A form that would put the example in the generated HTML pages for the dialect would be best, but at a minimum, some comment here would be helpful, especially if the tests aren't going to show the full CIR generated.